redaxo/src/core/lib/sql/sql.php

```diff
@@ -122,6 +122,7 @@ protected function selectDB($db)
                         PDO::MYSQL_ATTR_SSL_CA => $dbconfig->sslCa,
                     ];
                 }
+                $options[PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT] = $dbconfig->sslVerifyServerCert;
```

Comment:
Aktuell setze ich die Option immer. Oder ist sie nur sinnvoll in Kombination mit denen oben drüber?